libs.md

```diff
@@ -13,7 +13,7 @@ description: "The latest news about Amazon Ion and the Amazon Ion community."
 |------|----------------|------|---------------|
 | ion-c | [1.0.2](https://github.com/amzn/ion-c/releases/latest) (November 19, 2018) | [Link](https://github.com/amzn/ion-c) | [Link](https://amzn.github.io/ion-c/) |
 | ion-java | [1.5.0](https://github.com/amzn/ion-java/releases/tag/com_amazon_ion_v1.5.0) (June 26, 2019) | [Link](https://github.com/amzn/ion-java) | [Link](https://www.javadoc.io/doc/software.amazon.ion/ion-java/) |
```

Comment:
While you're in here, could you bump the `ion-java` version number to `1.6.0`?

Reply:
Checked others as well, bumped to the latest version where the page was out-of-date.